packages/citizen-sdk/src/utils/triggerFaucet.ts

```diff
@@ -81,6 +81,8 @@ export async function triggerFaucet({
 }: TriggerFaucetParams): Promise<TriggerFaucetResult> {
   // Throttle via localStorage (browser env only)
   if (typeof localStorage !== "undefined") {
+    alert("TX needs to be signed in order to claim.")
```

Comment:
We don't force UI behavior through the sdk.
sdk is only utility and error/message feedback.

front-end a dev can do with the messaging what they want or need to do.

Reply:
what do we wanna do instead in the sdk to notify user on that they need to sign a transaction @L03TJ3

Reply:
done